webapp/app/js/services/jobfilters.js

```diff
@@ -486,7 +486,8 @@ treeherder.factory('thJobFilters', [
     var _matchesDefaults = function(field, values) {
         $log.debug("_matchesDefaults", field, values);
         if (DEFAULTS.hasOwnProperty(field)) {
-            return _.intersection(DEFAULTS[field].values, values).length === DEFAULTS[field].values.length;
+            return values.length === DEFAULTS[field].values.length &&
```

Comment:
without this, we weren't noticing that you'd added ``coalesced`` to the global filters, so it wasn't showing them, due to the way ``_.intersection`` works.